sql/show.go

```diff
@@ -48,6 +48,9 @@ func (p *planner) Show(n *parser.Show) (planNode, error) {
 		v.rows = append(v.rows, []parser.Datum{parser.NewDString(p.txn.Proto.Isolation.String())})
 	case `TRANSACTION PRIORITY`:
 		v.rows = append(v.rows, []parser.Datum{parser.NewDString(p.txn.UserPriority.String())})
+	case `PANIC`:
+		// TODO(peter): Remove. This is just for testing.
```

Comment:
I'm going to remove this before merging this PR. I wanted to be able to force a panic during SQL execution in order to test the panic handling. This allowed me to create a panic by executing the statement `SHOW PANIC`.